CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## v1
 
+## v1.3.1
```

Comment:
As this change is not a bug fix, but a backward compatible change, I'm bumping the versioning

```suggestion
## v1.4.0
```